drv/lpc55-update-server/src/main.rs

```diff
@@ -432,7 +403,7 @@ impl idl::InOrderUpdateImpl for ServerImpl<'_> {
             return Err(UpdateError::UpdateInProgress.into());
         }
         self.syscon.chip_reset();
-        panic!()
+        unreachable!();
```

Comment:
Did this reduce the flash size? according to https://doc.rust-lang.org/stable/std/macro.unreachable.html this will still produce an implicity panic